app/src/main/java/fr/free/nrw/commons/customselector/ui/selector/CustomSelectorViewModel.kt

```diff
@@ -0,0 +1,46 @@
+package fr.free.nrw.commons.customselector.ui.selector
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.MutableLiveData
+import fr.free.nrw.commons.customselector.listeners.ImageLoaderListener
+import fr.free.nrw.commons.customselector.model.CallbackStatus
+import fr.free.nrw.commons.customselector.model.Image
+import fr.free.nrw.commons.customselector.model.Result
+
+class CustomSelectorViewModel(application: Application) : AndroidViewModel(application) {
```

Comment:
You can make your ViewModels injectable using Dagger.

Reply:
On it.

Reply:
Done.